src/_pytest/runner.py

```diff
@@ -220,12 +231,22 @@ def call_runtest_hook(item, when: "Literal['setup', 'call', 'teardown']", **kwds
 
 @attr.s(repr=False)
 class CallInfo:
-    """ Result/Exception info a function invocation. """
+    """ Result/Exception info a function invocation.
+
+    :param excinfo: `ExceptionInfo` describing the captured exception if any
+    :param start: (float) The system time when the call started, in seconds since the epoch.
+    :param stop: (float) The system time when the call ended, in seconds since the epoch.
+    :param duration: (float) The call duration in seconds
+    :param duration_ns: (int) The call duration in nanoseconds. Only in python >= 3.7
+    :param when: context of invocation: "setup", "call", "teardown"...
+    """
 
     _result = attr.ib()
     excinfo = attr.ib(type=Optional[ExceptionInfo])
     start = attr.ib()
     stop = attr.ib()
+    duration = attr.ib()
```

Comment:
Would be good to add `type=` at least for the newly added attributes, for mypy's benefit.

Reply:
why not, yes. Not sure they are picked up by mypy when declared in `attrs` actually, but that might make it more readable.

Reply:
They are indeed picked up -- mypy has a special plugin for attrs.